src/Models/Constants/VisualRuleTypes.ts

```diff
@@ -0,0 +1,10 @@
+export interface VisualColorings {
+    color: string;
+    label: string | undefined;
+}
+
+export interface VisualBadges {
+    color: string;
+    iconName: string;
+    labelExpression: string | undefined;
```

Comment:
can we make it optional with `label?` instead of adding `undefined `as or value?